stable/rabbitmq/values.yaml

```diff
@@ -167,11 +174,8 @@ ldap:
   port: "389"
   user_dn_pattern: cn=${username},dc=example,dc=org
   tls:
+    # If you enabled TLS/SSL you need to use the advancedConfiguration parameter to set advanced options.
```

Comment:
Is this mandatory? Is it worth to add a validation for this?

Reply:
It is not mandatory. It depends on what you need to configure.
I reworded the sentence.